platform-hub-web/src/app/kubernetes/kubernetes-robot-tokens-list.html

```diff
@@ -67,7 +67,7 @@
           <strong>Cluster:</strong> {{t.cluster.name}}
         </p>
         <p class="md-body-1">
-          <strong>Token:</strong> {{t.token}}
+          <strong>Token:</strong> <kubernetes-token item="t"/>
```

Comment:
Deprecated technology alert 😛  – need:

```
<kubernetes-token item="t"></kubernetes-token>
```

(I think the final build will fail otherwise, in certain cases).